packages/scoutgame-ui/src/components/common/Navigation/Header.tsx

```diff
@@ -169,7 +169,7 @@ export function Header() {
                       <InfoIcon color='secondary' />
                     </IconButton>
                   </Link>
-                  {platform === 'webapp' && (
+                  {(platform === 'webapp' || platform === 'onchain_webapp') && (
```

Comment:
might be good to add a "isWebappPlatform()" method that checks for either platform

Reply:
yeah this is the only place afaik